pyof/v0x01/controller2switch/stats_request.py

```diff
@@ -44,29 +48,20 @@ def pack(self):
         Make `body` a binary pack before packing this object. Then, restore
```

Comment:
Please, remember that the `pack` method must have a `value` attribute on the signature.